Verified Domains default role (#536)
clerk/clerk-docs: docs/organizations/verified-domains.mdx

An application instance may only have one verified domain of the same name, and an organization may only have one domain of the same name (verified or unverified).

## Default role
When enabling the Verified domains feature, you are also required to define which will role will be assigned to the user that will receive the Organization Invitation or Organization Suggestion in order to join your Organization.
You can choose any of the available Organization Roles of your instance.

> Please keep in mind, that you can't delete an Organization Role if it's used as the Verified Domain default role.

Reviewer: This should be a "warning":

```suggestion
<Callout type="warning">
You can't delete an Organization Role if it's used as the Verified Domain default role.
</Callout>
```

> You can update this setting via Clerk Dashboard.

Reviewer: Provide a link to the appropriate place to perform this operation in the Clerk Dashboard [following the guidance in the styleguide](https://github.com/clerk/clerk-docs/blob/main/STYLEGUIDE.md#410-provide-users-with-clear-instructions-and-a-direct-link-when-navigating-the-clerk-dashboard).

Author: Updated!
